handlers/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "handlers"
+version = "0.1.0"
+authors = ["Intent HQ <engineering@intenthq.com>"]
+edition = "2018"
+
+[dependencies]
+git = { path = "../git" }
+actix-web = "0.7.*"
```

Comment:
As in the other PR, update these version ranges to be `X.Y.Z` or `^X.Y.Z` ranges, rather than use `*` or `X.Y` or just `X`.